ci/views.py

```diff
@@ -586,15 +590,36 @@ def identity(request, identity):
                         extra_tags='success'
                     )
 
+            elif 'optout_identity' in request.POST:
+                details = results.get('details', {})
+                addresses = details.get('addresses', {})
+
+                for address_type, addresses in addresses.items():
+                    for address, info in addresses.items():
+                        idApi.create_optout({
```

Comment:
There is the `api/v1/change/` endpoint that is common in both hellomama and momza. They both expect different things. Maybe this requires more thought.